docs/readthedocs/source/doc/Nano/QuickStart/pytorch_inference.md

```diff
@@ -186,30 +187,30 @@ There are a few arguments required only by INC, and you should not specify or mo
 Here is an example to use INC with accuracy control as below. It will search for a model within 1% accuracy drop with 10 trials.
 ```python
 from torchmetrics.classification import Accuracy
-trainer.quantize(model,
-                 precision='int8',
-                 accelerator=None,
-                 calib_dataloader= dataloader,
-                 metric=Accuracy()
-                 accuracy_criterion={'relative': 0.01, 'higher_is_better': True},
-                 approach='static',
-                 method='fx',
-                 tuning_strategy='bayesian',
-                 timeout=0,
-                 max_trials=10,
-                ):
+InferenceOptimizer.quantize(model,
+                            precision='int8',
+                            accelerator=None,
+                            calib_dataloader= dataloader,
+                            metric=Accuracy()
+                            accuracy_criterion={'relative': 0.01, 'higher_is_better': True},
+                            approach='static',
+                            method='fx',
+                            tuning_strategy='bayesian',
+                            timeout=0,
+                            max_trials=10,
+                            ):
 ```
 **Accuracy Control with POT**
 Similar to INC, we can run quantization like:
 ```python
 from torchmetrics.classification import Accuracy
-trainer.quantize(model,
-                 precision='int8',
-                 accelerator=`openvino`,
-                 calib_dataloader= dataloader,
-                 metric=Accuracy()
-                 accuracy_criterion={'relative': 0.01, 'higher_is_better': True},
-                 approach='static',
-                 max_trials=10,
-                ):
+InferenceOptimizer.quantize(model,
+                            precision='int8',
+                            accelerator=`openvino`,
+                            calib_dataloader= dataloader,
+                            metric=Accuracy()
+                            accuracy_criterion={'relative': 0.01, 'higher_is_better': True},
+                            approach='static',
+                            max_trials=10,
+                            ):
```

Comment:
other than these changes, we may add a warning note to tell our previous users(if any), Trainer's API will be deprecated in futyre.

Reply:
done.